docs/STAGING_ENVIRONMENTS.md

```
@@ -0,0 +1,337 @@
+  -H "Content-Type: application/json" \
+  -d '{"query": "{ __schema { queryType { name } } }"}'
+```
+
+## Environment Variables Reference
+
+### Vercel (Frontend)
+
+| Variable | Development (Local) | Staging | Production |
+|----------|-------------------|---------|------------|
+| VITE_ENVIRONMENT | development | staging | production |
+| VITE_GRAPHQL_ENDPOINT | http://localhost:8000/graphql | https://gymintel-backend-staging.railway.app/graphql | https://gymintel-backend.railway.app/graphql |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)